src/Compilers/Core/Portable/SourceGeneration/GeneratorContexts.cs

```diff
@@ -191,7 +191,7 @@ public void RegisterForSyntaxNotifications(SyntaxContextReceiverCreator receiver
         public void RegisterForPostInitialization(Action<GeneratorPostInitializationContext> callback)
         {
             CheckIsEmpty(InfoBuilder.PostInitCallback);
-            InfoBuilder.PostInitCallback = callback;
+            InfoBuilder.PostInitCallback = (context) => callback(new GeneratorPostInitializationContext(context.AdditionalSources, CancellationToken.None));
```

Comment:
>(context) [](http://example.com/codeflow?start=43&length=9)

Should the `PostInitCallback` have a `CancellationToken` parameter? (Just curious.)

Reply:
Yeah, we used to have the token, but it never seemed necessary. I removed it in the newer APIs, and here its just for back compat. That being said, given that we're *adding* tokens everywhere else, I'll bring it back into the new API and pass it through.